### DISCUSSION TITLE
antd less integration not working

### DISCUSSION THREAD

**Discussion by mtahagocer** [vercel/next.js#27020] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

16.4.1

### What browser are you using?

chrome

### What operating system are you using?

windows

### How are you deploying your application?

vercej

### Describe the Bug

I used sass options bot not working  with less

{  cssModules: true,
  sassOptions: {
    includePaths: [path.join(__dirname, 'styles')],
  },

```
yarn dev
yarn run v1.22.10
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from D:\Projects\powerdev-frontend\.env.local      
info  - Loaded env from D:\Projects\powerdev-frontend\.env.development
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
error - ./styles/antd.less
Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> @import "~antd/dist/antd.less";
|
| @primary-color: #000; // primary color for all components
```

### Expected Behavior

I tryed all zeit-css modules but not worked on next 11

### To Reproduce

yarn dev

**Comment by sachinraja** [comment]
There is no built-in less loader. You can try [this plugin](https://github.com/elado/next-with-less).

**Reply by Me-UserName** [reply]
http://www.tigerhu.com/article/js/27381.html
